aksel.nav.no/website/sanity/schema/documents/presets/editors.ts

```diff
@@ -8,4 +9,32 @@ export const editorField = defineField({
   of: [{ type: "reference", to: [{ type: "editor" }] }],
   group: "settings",
   validation: (Rule) => Rule.required(),
+  initialValue: async (_, { currentUser, getClient }) => {
+    const client = getClient({ apiVersion: SANITY_API_VERSION });
+    let profile = await client.fetch(
+      `*[_type == "editor" && (lower($mail) == lower(email) || lower($mail) == lower(alt_email))][0]`,
+      { mail: currentUser?.email },
+    );
+
+    /**
+     * If user forgot to create a profile, we create one for them
+     */
+    if (!profile) {
+      /* Make sure to only create profile for SSO-logins */
+      if (
+        !currentUser?.email.endsWith("@nav.no") ||
+        !currentUser.provider?.includes("saml")
+      ) {
+        return [];
+      }
+      profile = await client.createIfNotExists({
+        _type: "editor",
+        _id: `auto-editor.${currentUser?.id}`,
+        email: currentUser?.email,
+        title: currentUser?.name,
```

Comment:
Trenger for så vidt ikke `?` her
```suggestion
        _id: `auto-editor.${currentUser.id}`,
        email: currentUser.email,
        title: currentUser.name,
```